Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1

```diff
@@ -41,6 +41,8 @@ Write-Host "Package= $Package"
 Write-Host "AdditionalArguments= $AdditionalArguments"
 
 #Find the package to deploy
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
```

Comment:
this task was working without the import specifically. Is it because its path was added to the PSModule path? Why was this needed post your change?